Final_project_2/frontend/css/style.css

```diff
@@ -485,26 +480,34 @@ address{
   margin-left: 80px;
   margin-bottom: 70px;
 }
+
 address .fa {
   display: block;
   font-size: 33px;
   margin-bottom: 20px;
 }
 
+.address-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
```

Comment:
this is the property I mentioned above that was previously on the `fanmail` class; but since in practice only these 3 guys needed it, I moved it here.